benches/ironoxide_bench.rs

```diff
@@ -119,80 +118,70 @@ fn criterion_benchmark(c: &mut Criterion) {
         group_enc_deks,
         group1,
         group2,
-    ) = rt.enter(|| block_on(f));
+    ) = rt.block_on(f);
 
     c.bench_function("document get metadata", |b| {
-        b.iter(|| rt.enter(|| block_on(io.document_get_metadata(black_box(enc_result.id())))))
+        b.iter(|| rt.block_on(io.document_get_metadata(black_box(enc_result.id()))))
```

Comment:
I don't remember why we used the `rt.enter` syntax in the first place, so if this change to `rt.block_on` is worse/wrong, please let me know

Reply:
I'm not sure either. I assume it didn't work the other way anymore?

Reply:
They at least changed it enough that it wouldn't compile as-is, but I didn't try too hard to make it work the same since `block_on` seemed more like what I expected anyway